### PR TITLE
Integrate MR GitHub issues into ODH dashboard

### LINKED ISSUE #53: Integrate Model Registry issues into ODH dashboard
Need to integrate Model Registry issues in the ODH dashboard. For that automatically take place we need to add the following [YAML file](https://github.com/opendatahub-io/notebooks/blob/main/.github/workflows/auto-add-issue-to-project.yml) in the repo

### PR DESCRIPTION
**Pull request by tarilabs**
Resolves #53 

## Merge criteria:

- [x] The commits and have meaningful messages; the author will squash them [after approval](https://github.com/opendatahub-io/opendatahub-community/blob/main/contributor-cheatsheet.md#:~:text=Usually%20this%20is%20done%20in%20last%20phase%20of%20a%20PR%20revision) or will ask to merge with squash.
- [n/a] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [n/a] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: .github/workflows/auto-add-issue-to-project.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+name: Add Issues to  Model Registry ODH board Project
```
Comment (lampajr) [inline .github/workflows/auto-add-issue-to-project.yml]: nitpick :)
```suggestion
name: Add Issues to Model Registry ODH board Project
```

File: .github/workflows/auto-add-issue-to-project.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    types:
+      - opened
+jobs:
+  add-to-project:
+    name: Add issue to project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@v0.5.0
+        with:
+          project-url: https://github.com/orgs/opendatahub-io/projects/49
```
Comment (lampajr) [inline .github/workflows/auto-add-issue-to-project.yml:13]: @rareddy do we want to keep it private? or is it just temporary solution?

**Review by lampajr — CHANGES_REQUESTED** [review]

File: .github/workflows/auto-add-issue-to-project.yml (hunk 2 above)
Comment (rareddy) [reply .github/workflows/auto-add-issue-to-project.yml:13]: temporary, working to make it public

**Review by rareddy — APPROVED** [review]
/lgtm

**Review by lampajr — APPROVED** [review]

File: .github/workflows/auto-add-issue-to-project.yml (hunk 2 above)
Comment (rareddy) [reply .github/workflows/auto-add-issue-to-project.yml:13]: it should be public now, and issues are showing up, Yay!

Comment (tarilabs) [reply .github/workflows/auto-add-issue-to-project.yml:13]: @rareddy I've manually added some old ones by hand with all my love 😅

Comment (tarilabs) [reply .github/workflows/auto-add-issue-to-project.yml:13]: the GHA will ensure all new ones will be automatically linked in the board project 👍

**Pull request merged**